tajo-core/src/main/java/org/apache/tajo/engine/planner/PhysicalPlannerImpl.java

```diff
@@ -854,6 +855,14 @@ private PhysicalExec createSortBasedColumnPartitionStorePlan(TaskAttemptContext
           sortSpecs[i++] = new SortSpec(insertNode.getProjectedSchema().getColumn(id), true, false);
         }
       }
+    } else if (storeTableNode.getType() == NodeType.CREATE_TABLE) {
+      int i = 0;
+      for (int j = 0; j < partitionKeyColumns.length; j++) {
+        int id = storeTableNode.getOutSchema().getColumns().size() + j;
```

Comment:
In my opinion, we should consider the qualified name for CTAS with defined column. 
@blrunner @hyunsik , How do you think?